### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by gillchristian**
This PR adds:

- Support for setting app port and mongodb uri via env vars.
- Dockerize the generated app
    - Run the app in a [node:wheezy](https://hub.docker.com/_/node/) container.
    - Run [mongodb](https://hub.docker.com/_/mongo/) in a container.
    - Persist data using [tianon/true](https://hub.docker.com/r/tianon/true/) container.
    - Run everything with docker-compose.

### CODE REVIEW

File: generators/app/templates/config.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ const milieu = require('milieu');
 const config = milieu('<%= serverName %>', {
   environment: 'dev',
   server: {
-    port: 8080
+    port: process.env.PORT || 8080
```
Comment (ndelvalle) [inline generators/app/templates/config.js]: milieu package allow you to use templates like:

```javascript
port: '${PORT}' || 8080
```

File: generators/app/templates/config.js [hunk 2]
```diff
@@ -3,10 +3,10 @@ const milieu = require('milieu');
...
   },
   mongo: {
-    url: 'mongodb://localhost/<%= databaseName %>'
+    url: process.env.MONGO_DB_URI || 'mongodb://localhost/<%= databaseName %>'
```
Comment (ndelvalle) [inline generators/app/templates/config.js]: Same as the above comment

File: generators/app/templates/config.js (hunk 1 above)
Comment (gillchristian) [inline generators/app/templates/config.js]: Cool, will update to it

**Comment by ndelvalle** [comment]
LGTM 👍

**Pull request merged**